accessify/utils.py

```diff
@@ -244,3 +244,19 @@ def get_class_members(class_):
             inspected_members[member.unique_name] = member
 
     return inspected_members
+
+
+def get_method_class(frame):
```

Comment:
flake8 complains about it: `D103 Missing docstring in public function`
https://travis-ci.com/dmytrostriletskyi/accessify/jobs/186522427

Reply:
@dtitov, done.